src/vos/tests/vts_io.c

```diff
@@ -682,6 +687,10 @@ io_update_and_fetch_dkey(struct io_test_args *arg, daos_epoch_t update_epoch,
 	iod.iod_nr	= 1;
 
 	/* Act */
+	if (fault_injection_flag == FAULT_INJECT) {
+		daos_fail_loc_set(DAOS_CHECKSUM_UPDATE_FAIL | DAOS_FAIL_ALWAYS);
+		daos_fail_num_set(2);
```

Comment:
same here, what is this for?

Reply:
Same comment:

Based on my testing, it is the number of times fault injection flag is tested.
eg: If we perform DAOS_FAIL_CHECK once, the fail_number is decremented by 1.
In multiple extent testing, we are doing the DAOS_FAIL_CHECK once. In the single value testing, we are doing DAOS_FAIL_CHECK at twice.